pkg/model/types.go

```diff
@@ -40,10 +40,18 @@ const (
 
 type ServiceType string
 
+const (
+	IPV4Type IPType = "IPV4Type"
+	IPV6Type IPType = "IPV6Type"
+)
+
+type IPType string
```

Comment:
I'm don't think this type is required, the type can be derived from IP address, so function like 
```func (e *Endpoint) GetIPAddressType() string ``` should be enough.  So, all the code branching around this type can be abstracted out.

Reply:
It's removed, thanks.